### PR TITLE
Demo Submisson: Multi-account AWS deployment pipeline for a ToDo app

### PR DESCRIPTION
**Pull request by andrebrogard**
The link to the video can be found [here](https://youtu.be/morsy_tXLYE).

This follows proposal #1052

### CODE REVIEW

**Comment by SophieHYe** [comment]
Feedback:

## Good points
* The voice is very good and spoken clearly.
* The UI of the web app is good.
* Good to have subtitles.


## To improve
* Expand the introduction. More background could be given, e.g., cloudformation and To-do app.
* The pace is a bit too fast.
* The font size of the code/text is too small, audience hard to follow.
* The demo is lacking of a focus. Too much info are putting in a short demo.
* Please consider to expand the motivation and give a clear take away.
* Not perfectly timing within 3 minutes.

**Comment by mans-andersson** [comment]
# Demo day feedback

## Summarizing thoughts
Below I have described the flaws that I could find in the demo
but I think that the overall impression is good and 
professional. The voiceover is good and I feel like the
instructions were very clear even though the tempo is
very high (which is of course explained by the 3 minute limit).
I had to raise the volume on my computer quite high to hear the
voiceover clearly. Perhaps it would be possible to increase the 
voice volume a bit in the video editing software. Good job!

## Multi-account
For me it was a little bit unclear what part of 
the tutorial makes this multi-account. And what exactly does
multi-account mean? Multiple AWS accounts? If that is the 
case then what is good about a multi-account deployment?
I do not have a lot of experience with cloud development/AWS
so that may be the reason this is not clear for me.

## Length
The video has the length 3:22 which is more than 3 minutes. 
Maybe this is fine but I think I read somewhere earlier that 
the 3 minute limit is very strict.

## Youtube description
The description for the video is currently empty. 
It should at least contain the link to the code repo
which is also mentioned at the end of the video.

**Comment by damnePers** [comment]
Thank you for the demo, here is some feedback:

First of all, your demo is too long. Time 3:22, shorten this to get to 3 minutes since that was a strict limit. 

It is good that you start with what the demo is about and what you have done as prerequisites. Also, it is nice that you show and define your pipeline which gives the viewer a good overview of what's to come. However, this is your main focus, it shouldn’t be. Your video feels more like a presentation about the pipeline than an actual demo of how to set it up and use it. I would suggest that you add, in the first part - up until 2:17, more actionable steps. Instead of just showing code, show where to add it, how to write it/get it etc. Show how to 

Regarding the beginning part where you have slides with a lot of text as you speak, it is really difficult to follow along in the video as well as reading the slides. Here is a good part where you could switch out the slides for actionable steps instead. 

Emphasis what part about your demo actually integrates multi.account. As I see it you only show how to set up a pipeline and not how to connect multiple accounts. 

It would be nice if there were some documentation where you list what you are doing in the demo so that the viewer could easily access the commands and code used in the demo. There you could also list what prerequisites are needed for this demo and instructions for the steps you make in your demo so the viewer could look up any details they might have missed or find difficult. 

Also, link to the example repo you mention in your proposal.

**Comment by cpiehl1** [comment]
Nice demo! I had a very good overall impression. Voice over was clear and the tempo was good. Visually quite pleasing, especially the AWS console parts. However, the code snippets were a bit overwhelming, highlighting certain parts would probably help.

It seems like the technical content is quite extensive and well executed. However, I do not really see a strong enough connection between the multi-account aspect and the pipeline aspect. I feel like including both widens the scope a bit too much for a 3 minute demo. Or maybe it just needs to be motivated a bit more.

As others have said, the video is a bit too long.

A lot of concepts are mentioned without much background

**Comment by George-Bassilious** [comment]
Great Demo! Very informative and easy to understand! I have written down some points I thought of while watching the video:

## Positives
* Very informative video! I really enjoyed the structure and felt that I really learned something new
* I really enjoyed following the process and seeing the different steps which is perfectly linked with the background info given.
* Very nice and well placed diagrams that helped me understand the process better
* Great that you added closed captions, did not find them at first however I like that you gave the viewers the option to have subtitles or not


## Possible suggestions for improvement
* The length of the Demo is way to long, It should be 3 minutes and now it is 20 seconds too long, perhaps you can speed up the video or get rid of some the pauses that exist eg: 2.55 , 3.02-3.05 etc.
* It would be really nice if you added the github repo to the description and maybe add some more info there to give the video demo some more life, even though its not a requirement I feel it adds more to the interactivity with the viewer.
* Something I like in Demo videos is background music, so a tip is to add some low tunes in the background to increase immersion but I understand if you didn´t want to have that.
* sometimes it was difficult to hear the voice over especially at the ending bits, and the closed caption were off sync at around 3:10, that made it difficult to follow along at the end and i had to rewind to really understand the ending parts

**Comment by evkade** [comment]
**Demo Feedback**

Good and interesting video with a relevant subject! 

_Positive_ 
- Nice that you show the app the demo before so we have a clear context 
- Nice with some explanations in the first slide to relate it to devops 
- Nice to have the infrastructure and deployment diagrams 
- The video is clear and has an interesting subject
- The instructions are easy to follow 
- The repo looks good and complete 

_Negative_
- The video is too long 
- Maybe the slides in the beginning could have some kind of illustrations or schemes to illustrate processes 
- Sometimes it feels like the video is going a little bit slow 
- I think the repo should be linked in the description so we don’t have to type in the link

**Comment by majate** [comment]
Good job with completing the demo on your own! The video had a good introduction that explained the context and gave a motivation to why the content of the video could be useful. The voice over was clear and had a slow and steady pace, and the subtitles were correct. Some specific comments:

- Note that to pass the Demo task, your video cannot exceed 3 minutes. Your video is currently 3:22, so you will have to reduce it by 22 seconds. I think that the voice over could be speed up a tiny bit without being too rushed, if you think that is a solution. Otherwise you may need to find parts to cut.
- The slides have in some places a lot of text on them. Text can be a good way to include additional information in the video, without needing to mention it explicitly. However, seeing a lot of text without being given the time to read it, can make viewers feel stressed. To make it easier for us viewers to know which information is important, it may be good visually highlight the important parts. Currently you have some slides where you mention some bullets, but don't mention others. This made me initially think that I had missed something that you said. By visually differentiating the information that is mentioned in the video, and what is only additional information may increase the quality of the video. To do so you can modify the font, font size, color, style etc.
- Around timestamp 3:10, the subtitle is a bit off (one phrase comes too late).
- This is not really a comment about the video itself, but including a description in Youtube and providing the URL to your to-do repo there would increase the quality of the video if you want to publish it publicly.

Good luck with finalizing the video!
